conda/environments/cugraph_dev_cuda10.yml

```diff
@@ -3,8 +3,10 @@ channels:
 - rapidsai/label/cuda10.0
 - nvidia/label/cuda10.0
 - rapidsai-nightly/label/cuda10.0
+- rapidsai/label/thirdparty
 - conda-forge
 dependencies:
+- libcypher-parser
```

Comment:
`libcypher-parser` should also be in 10.1 dev env file once it is merged.

You probably need to update `build.sh` under ci and `meta.yml` under conda.